### ISSUE TITLE
Character sheet causing random crashing on button click.

### ISSUE THREAD

**Issue by apewall** [free-beer/bh2e#35]
When clicking any of the orange buttons on the character actor sheet sometimes a failure will occur which will open a new browser tab. This happens with both Chrome and Firefox. 

Nothing appears to be recorded in the console log and the intended click action never occurs.  I've had no luck tracking own the exact cause thus far myself.

https://user-images.githubusercontent.com/1915182/226471347-973f872c-fc5f-4545-85a7-3b201cb63da7.mp4

**Comment by free-beer** [comment]
Okay, I think your video shows the cause. You'll note that when you clicked it opened a new tab in your browser. This indicates that there's an anchor link somewhere on the sheet that doesn't have an event handler that cancels the default behaviour attached to it. The other thing I note is that you're having to click back and forth between action icons to trigger this. It might be surprising but when you click on these action icons you're updating the character data model which causes the character sheet to be re-rendered. I wonder if doing this more slowly reproduces the same result?

**Comment by free-beer** [comment]
Alright, a bit of experimentation shows that this isn't quite what I thought it was. I tried reproducing it using the method in your video and it just wasn't happening. I then noticed that on your last click you when a little high on the link, into the surrounding glow, and this appears to be where the unmitgated link opening is coming from. It's also produces the result more reliably. Will take a look to see if there is something anomalous in the markup that might be causing this.

**Comment by free-beer** [comment]
Alright, this turned out to be me putting the click event handler on the icon rather than on the surrounding anchor element. When you moused right to the edge of the glow you were clicking the anchor rather than the icon and that didn't have an event handler attached so it tried to follow the link, which opens the new tab. I should really remove the surrounding anchor element as it's technically not needed but I've just added a do nothing event handler to these elements as a quick kludge and will look to tidy these up later. Fix has been merged into the repository and will go out with the next release.
